package.json

```diff
@@ -30,21 +33,19 @@
     }
   ],
   "scripts": {
-    "audit": "lerna run audit --stream && npm audit --production",
-    "bootstrap": "lerna bootstrap",
-    "bump": "lerna publish",
+    "audit": "npm audit --production",
```

Comment:
This is so cool and simple!
It automatically audits every package in our workspace!